integreat_cms/static/src/js/poi-categories/poicategory-colors-icons.ts

```diff
@@ -0,0 +1,51 @@
+const renderColorPreview = ({ target }: Event) => {
+    const colorSelect = target as HTMLSelectElement;
+    const colorPreview = document.getElementById("poi-category-color-preview");
+    const selectedColor = colorSelect.options[colorSelect.selectedIndex].value;
+    console.debug(selectedColor);
+    if (selectedColor) {
+        colorPreview.style.backgroundColor = selectedColor;
+        colorPreview.classList.remove("hidden");
+        colorSelect.classList.add("rounded-none", "rounded-r", "border-l-0");
+    } else {
+        colorPreview.classList.add("hidden");
+        colorSelect.classList.remove("rounded-none", "rounded-r", "border-l-0");
+    }
+};
+
+const renderIconPreview = ({ target }: Event) => {
+    const iconSelect = target as HTMLSelectElement;
+    const iconPreview = document.getElementById("poi-category-icon-preview");
+    const selectedIcon = iconSelect.options[iconSelect.selectedIndex].value;
+    iconPreview.setAttribute("src", `${iconPreview.dataset.basePath}${selectedIcon}.svg`);
+    console.debug(selectedIcon);
+
+    if (selectedIcon) {
+        iconPreview.classList.remove("hidden");
```

Comment:
```suggestion

    if (selectedIcon) {
        iconPreview.setAttribute("src", `${iconPreview.dataset.basePath}${selectedIcon}.svg`);
        iconPreview.classList.remove("hidden");
```
@seluianova good catch!